parsl/monitoring/logging_server.py

```diff
@@ -0,0 +1,42 @@
+import tornado.ioloop
+import tornado.web
+import json
+from parsl.monitoring.db_logger import get_db_logger
+
+
+class MainHandler(tornado.web.RequestHandler):
+    """ A handler for all requests/logs sent to the logging server."""
+    def initialize(self, db_logger_config_object):
+        """This function is called on every request which is not ideal but __init__ does not appear to work."""
```

Comment:
what does that mean?

Reply:
With my attempts, I could not get __init__ to work with this handler in order to pass in and set the config information. Instead, there is this initialize method that to my understanding is called at the beginning of every HTTP request using the arguments from the request and additional arguments specified during the Applications configuration and start. In this sense, it is not ideal to have to keep setting something that stays constant.